base/process/internal_linux.cc

```diff
@@ -86,28 +92,29 @@ bool ParseProcStats(const std::string& stats_data,
   // pid (process name) data1 data2 .... dataN
   // Look for the closing paren by scanning backwards, to avoid being fooled by
   // processes with ')' in the name.
-  size_t open_parens_idx = stats_data.find(" (");
-  size_t close_parens_idx = stats_data.rfind(") ");
-  if (open_parens_idx == std::string::npos ||
-      close_parens_idx == std::string::npos ||
-      open_parens_idx > close_parens_idx) {
+  size_t pid_end = stats_data.find(" (");
+  size_t comm_start = pid_end + 2;
+  size_t comm_end = stats_data.rfind(") ");
+  size_t state_start = comm_end + 2;
+  if (pid_end == std::string::npos ||
+      comm_end == std::string::npos ||
+      pid_end > comm_end) {
```

Comment:
Instead of modifying the code in place, shall we gate the change by 
```#if defined(Starboard)?```
It will be easier for use to identify what customization we have made.

Also, it seems like if we keep some variable name the same, eg open_parens_idx, we may keep our code change minimum?